### DISCUSSION TITLE
column with images

### DISCUSSION THREAD

**Discussion by sbritz** [rappasoft/laravel-livewire-tables#654] [Q&A] [answered]
Hi,
first thx for this great datatable-package!!! Awesome work!
I have got one question: Is it possible to make a column for images like profile photo for example? Or is it only for clean text tables?
I've nothing found in the documentation.

**Comment by fabio-ivona** [comment, marked as answer]
you can use the ->format() method during columns definition and pass the view you'd like to render in each cell

**Comment by sbritz** [comment]
Cool, thx. It works! :-)
This is my solution

<img width="552" alt="Bildschirmfoto 2022-02-13 um 17 15 29" src="https://user-images.githubusercontent.com/25327081/153762186-e838bff9-2545-438b-b0c7-72a649a9f6be.png">

**Reply by kmaphane** [reply]
Thanks for this. but instead of asHtml I had to just html.
